src/main/java/com/jagrosh/discordipc/IPCClient.java

```diff
@@ -180,127 +289,186 @@ public final ClientBuild getClientBuild()
     // Enums
     
     /**
-     * Client status
+     * Constants representing various status that an {@link IPCClient} can have.
      */
     public enum Status
     {
-        CREATED, CONNECTED, CLOSED, DISCONNECTED
-    }
-    
-    /**
-     * Discord builds
-     */
-    public static enum ClientBuild
-    {
-        ANY, STABLE, PTB, CANARY;
-        
-        public static ClientBuild from(String endpoint)
-        {
-            switch(endpoint)
-            {
-                case "//canary.discordapp.com/api": return CANARY;
-                case "//ptb.discordapp.com/api": return PTB;
-                case "//discordapp.com/api": return STABLE;
-                default: return ANY;
-            }
-        }
+        /**
+         * Status for when the IPCClient has been created.<p>
+         *
+         * All IPCClients are created starting with this status,
+         * and it never returns for the lifespan of the client.
+         */
+        CREATED,
+
+        /**
+         * Status for when the IPCClient is connected with Discord.<p>
+         *
+         * This is only present when the connection is healthy, stable,
+         * and reading good data without exception.<br>
+         * If the environment becomes out of line with these principles
+         * in any way, the IPCClient in question will become either
+         * {@link Status#DISCONNECTED}.
```

Comment:
> will become either disconnected

... or closed?

Reply:
The `CLOSED` is inline more with `CONNECTED` as it's a healthy and normal response to the `CLOSE` OpCode.

When the connection is not good, it will become `DISCONNECTED`